src/App.js

```diff
@@ -112,8 +93,9 @@ export default function App() {
     inputValueInfo,
     img
   ) {
+    const id = nanoid();
     const newPlant = {
-      _id: nanoid(),
+      _id: id,
```

Comment:
Now you could rewrite it again to the way it was before:
```js
_id: nanoid();
```
Just a minor thought and you don't have to :)